readme.md

```diff
@@ -156,7 +154,7 @@ Next, we'll set up our local environment variables. We use [django-dotenv](https
 to help with this. It reads environment variables located in a file name `.env` in the top level directory of the project. The only variable we need to start is `DJANGO_SETTINGS_MODULE`:
 
     cp bakerydemo/settings/local.py.example bakerydemo/settings/local.py
-    echo "DJANGO_SETTINGS_MODULE=bakerydemo.settings.local" > .env
+    touch .env
```

Comment:
This is incomplete. After doing this, the developer is left with an empty `.env` and trying to run any management command will error out because `DJANGO_SETTINGS_MODULE` is unset, leaving them unable to continue. See #398.

Shouldn't this be something like

```suggestion
    echo "DJANGO_SETTINGS_MODULE=bakerydemo.settings.dev" > .env
```

instead?

Reply:
`DJANGO_SETTINGS_MODULE` should always be an optional environment variable, as it's set to that in `manage.py`. The `dev` variant is the default.

The main issue is that `local.py` is no longer considered a settings entrypoint in itself. Instead, it's automatically loaded when `.dev` is loaded. Previously, it was just an alias for .dev`. I've submitted a fix as https://github.com/wagtail/bakerydemo/pull/399